test/scripts/e2e_subs/asset-misc.sh

```diff
@@ -102,6 +102,16 @@ fi
 # case 3: asset created with manager, reserve, freezer, and clawback different from the creator
 ${gcmd} asset create --creator "${ACCOUNT}" --manager "${ACCOUNTB}" --reserve "${ACCOUNTC}" --freezer "${ACCOUNTD}" --clawback "${ACCOUNTE}" --name "${ASSET_NAME}" --unitname dma --total 1000000000000 --asseturl "${ASSET_URL}"
 
+# case 3a: asset info should fail if reserve address has not opted into the asset.
+if $(${gcmd} asset info --creator $ACCOUNT --unitname dma); then
+    date '+asset-misc FAIL asset info should fail unless reserve account was opted in %Y%m%d_%H%M%S'
+    exit 1
+else
+    echo ok
```

Comment:
@algochoi How involved is it to assert the error is the expected error?

Sorry for the late remark.  If it's onerous, I'm willing to merge as is though it feels unsafe.

Reply:
I added an assert for the error message as well

Reply:
@algochoi Resolving - Thanks for taking in the feedback.